app/controllers/planning_applications_controller.rb

```diff
@@ -91,11 +91,11 @@ def assign
   end
 
   def confirm_validation
-    @planning_application.documents_validated_at ||= if @planning_application.closed_validation_requests.present?
-                                                       @planning_application.last_validation_request_date
-                                                     else
-                                                       @planning_application.created_at
-                                                     end
+    @planning_application.validated_at ||= if @planning_application.closed_validation_requests.present?
```

Comment:
Since you're touching this anyway, it might be nice to pull this logic into the model? Something like `PlanningApplication#default_validated_at`?

Reply:
That's a great suggestion! Here it is https://github.com/unboxed/bops/pull/722/commits/dc0bb753ce81856b895b5c80a07fcb63bf916694